src/Symfony/Bundle/FrameworkBundle/Resources/config/translation.xml

```diff
@@ -140,11 +141,5 @@
             <tag name="kernel.cache_warmer" />
             <argument type="service" id="Psr\Container\ContainerInterface" />
         </service>
-
-        <service id="translator_listener" class="Symfony\Component\HttpKernel\EventListener\TranslatorListener">
```

Comment:
Should we deprecate the service instead in case someone is relying on its presence?

Reply:
How can we deprecate without it conflicting with the new `LocaleAwareListener` ?

Reply:
Does it do any harm if both listeners are executed?

Reply:
Yeah you're right. There is no undesirable side effect beside setting the locale twice

Reply:
I removed the `translator_listener` service completely as keeping it would trigger deprecation notices. I don't think that's an issue.